src/uploader.js

```diff
@@ -63,26 +62,22 @@ const useWidget = (
   const attributes = useDeepMemo(() => propsToAttr(options), [options])
 
   useDeepEffect(() => {
-    if (!firstRender.current) {
-      window.UPLOADCARE_LOCALE = locale
+    if (locale) window.UPLOADCARE_LOCALE = locale
+    if (localePluralize) window.UPLOADCARE_LOCALE_PLURALIZE = localePluralize
+    if (localeTranslations) {
       window.UPLOADCARE_LOCALE_TRANSLATIONS = localeTranslations
-      window.UPLOADCARE_LOCALE_PLURALIZE = localePluralize
-
-      uploadcare.plugin((internal) =>
-        internal.locale.rebuild({
-          locale,
-          localeTranslations,
-          localePluralize
-        })
-      )
     }
 
-    firstRender.current = false
+    if (locale || localePluralize || localeTranslations) {
+      uploadcare.plugin((internal) => {
+        internal.locale.rebuild({ locale, localeTranslations, localePluralize })
```

Comment:
Some of the these props could be undefined. Since you're already reassigning the `window.UPLOADCARE_` globals as needed, would it work to just call `internal.locale.rebuild()` here?

Reply:
> would it work to just call internal.locale.rebuild() here?
no, it wouldn't

I sync `window.UPLOADCARE_` globals for effects tab, because it use them too
and `internal.locale.rebuild({ locale, localeTranslations, localePluralize })` updates `uploadcare-widget` settings

Reply:
Ok. In that case, if any of the props are undefined, will rebuild remove them from the settings?